libpod/runtime_ctr.go

```diff
@@ -195,6 +204,49 @@ func (r *Runtime) GetContainers(filters ...ContainerFilter) ([]*Container, error
 	return ctrsFiltered, nil
 }
 
+// LogOptions is used for the GetLogs function
+// holds flag params defined by the user
+type LogOptions struct {
+	Details   bool
+	Follow    bool
+	SinceTime time.Time
+	Tail      uint64
+}
+
+// GetLogs returns the logs of a container from the log file
+// log file is created when the container is started/ran
+func (r *Runtime) GetLogs(container string, logChan chan string, opts LogOptions) error {
+	r.lock.RLock()
+	defer r.lock.RUnlock()
+
+	if !r.valid {
+		return ErrRuntimeStopped
+	}
+
+	defer close(logChan)
```

Comment:
Should this be before the check above?  Does the user expect the channel to be closed when GetLogs returns?

Reply:
uh, not sure I just modified the code that was already written, which was like this.